agent/startup.sh

```diff
@@ -103,6 +103,12 @@ fi
 
 export CATTLE_AGENT_IP=${CATTLE_AGENT_IP:-$DETECTED_CATTLE_AGENT_IP}
 
+if [ -n "$CATTLE_PHYSICAL_HOST_UUID" ]; then
```

Comment:
I believe the python code would currently interpret a blank but set env var as being set and use a blank value as the uuid. We could alternatively change the python code to treat blank strings the same as unset env vars. I was worried about unintended side effects of that bc it's a general purpose function that gets the env var.
On Jan 20, 2015, at 8:21 AM, Darren <notifications@github.com<mailto:notifications@github.com>> wrote:


In agent/startup.sh<https://github.com/rancherio/rancher/pull/37#discussion-diff-23228414>:

> @@ -103,6 +103,12 @@ fi
>
>  export CATTLE_AGENT_IP=${CATTLE_AGENT_IP:-$DETECTED_CATTLE_AGENT_IP}
>
> +if [ -n "$CATTLE_PHYSICAL_HOST_UUID" ]; then


Does this need to be conditional? Can you just always pass it and it might be a zero length string?

-
Reply to this email directly or view it on GitHub<https://github.com/rancherio/rancher/pull/37/files#r23228414>.